client/src/controllers/ActionController.ts

```diff
@@ -46,6 +46,13 @@ export class ActionController extends Controller<
   declare continueValue: boolean;
   declare urlValue: string;
 
+  select() {
```

Comment:
I have opted to move this to the last method, where logical this ordering is alphabetical (but that's not enforced).

Reply:
Yes, this does make a lot of sense.